### PR TITLE
Better command line parsing, show the window on the right desktop

### LINKED ISSUE #197: Nagstamon-qt opening the window in the wrong desktop
Hi again,

I am using nagstamon-qt in systray mode.

When I click the icon, sometimes it does not open any window. If I then start cycling all the virtual desktops, I find the window opened in a different one.

Couldn't still determine which criteria it uses to choose the desktop, but often it does not chose the right one.

### PR DESCRIPTION
**Pull request by gtozzi**
This fixes #197 and prevents #198  from happening.

### CODE REVIEW

File: Nagstamon/QUI/__init__.py [hunk 1]
```diff
@@ -1210,6 +1210,16 @@ def show_window(self, event=None):
...
 
+                    # Using the EWMH protocol to move the window to the active desktop.
+                    if not platform.system() in ('Darwin', 'Windows'):
+                        winid = self.winId().__int__()
+                        deskid = self.ewmh.getCurrentDesktop()
+                        self.ewmh.setWmDesktop(winid, deskid)
+                        self.ewmh.display.flush()
+
+                    # makes the window manager switch to the desktop where this widget has appeared
+                    self.raise_()
```
Comment (HenriWahl) [inline Nagstamon/QUI/__init__.py]: Is the desktop really changing here? I intended to open the popup at the desktop where the user clicks the systray icon.

Comment (gtozzi) [inline Nagstamon/QUI/__init__.py]: Yes, this makes the pop-up appear at the desktop where the user clicks the systray icon.

Comment (HenriWahl) [inline Nagstamon/QUI/__init__.py]: Cool. i will check if it still works outside of XFCE :-)

**Comment by gtozzi** [comment]
It also works on my KDE notebook.

File: Nagstamon/QUI/__init__.py (hunk 1 above)
Comment (gtozzi) [inline Nagstamon/QUI/__init__.py]: It also works on my KDE notebook.

**Comment by HenriWahl** [comment]
I had to apply your changes only if in systray mode because floating statusbar mode did now behave strangely - see https://github.com/HenriWahl/Nagstamon/commit/ed23d25f8a315316b007afdb013f6bf6556016ed. Can you check if it still works as expected?

**Comment by gtozzi** [comment]
Sorry, my fault: I didn't realize that function is being called for other modes too.
Yes, my mod was meant only for systray mode.
Yes, it still works as expected for me.
;)

**Comment by HenriWahl** [comment]
Fixed now.

**Pull request merged**